packages/store/ts/config/v2/table.ts

```diff
@@ -159,13 +164,15 @@ export function resolveTable<input extends TableInput, scope extends Scope = Abi
   const label = input.label;
   const type = input.type ?? TABLE_DEFAULTS.type;
   const namespace = input.namespace ?? TABLE_DEFAULTS.namespace;
+  const namespaceLabel = input.namespaceLabel ?? input.namespace ?? TABLE_DEFAULTS.namespace;
```

Comment:
we might want to require `namespaceLabel` in this context, otherwise its kind of unclear if the namespace label should inherit the input namespace or the default namespace